contracts/contribute/Contributors.sol

```diff
@@ -484,7 +494,7 @@ contract Contributors is ERC721TokenReceiver {
         // Check for pre-registration or deployed service state
         if (state == IService.ServiceState.PreRegistration) {
             // If pre-registration - re-deploy service first
-            _reDeploy(serviceId, stakingInstance, multisig);
+            _reDeploy(serviceId, stakingInstance, multisig, false);
```

Comment:
Staking an unstaked, terminated and unbonded service does not require service update, as the user is responsible for service update outside of this scope.